website/www/site/content/en/documentation/patterns/ai-platform.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "AI Platform integration patterns"
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# AI Platform integration patterns
+
+This page describes common patterns in pipelines with Google Cloud AI Platform transforms.
+
+{{< language-switcher java py >}}
+
+## Getting predictions
+
+This section shows how to use [Google Cloud AI Platform Prediction](https://cloud.google.com/ai-platform/prediction/docs/overview) to make predictions about new data from a cloud-hosted machine learning model.
+ 
+[tfx_bsl](https://github.com/tensorflow/tfx-bsl) is a library with a Beam PTransform called `RunInference`. `RunInference` is able to perform an inference that can use an external service endpoint for receiving data. When using a service endpoint, the transform takes a PCollection of type `tf.train.Example` and, for every batch of elements, sends a request to AI Platform Prediction. The size of a batch may vary. For more details on how Beam finds the best batch size, refer to a docstring for [BatchElements](https://beam.apache.org/releases/pydoc/current/apache_beam.transforms.util.html?highlight=batchelements#apache_beam.transforms.util.BatchElements).
```

Comment:
`tf.train.Example` -> `tf.train.Example` or `tf.train.SequenceExample`

Reply:
`tf.train.SequenceExample` is not supported for remote prediction: [link](https://github.com/tensorflow/tfx-bsl/blob/a49260a3935ef0f35456ec119a827b158eba3b2a/tfx_bsl/beam/run_inference.py#L406)

Reply:
@rose-rong-liu - any suggestions here?

Reply:
We're actively looking for solutions for sequence example support, which we may leverage arrow on. Maybe add a note to mention sequence example will be supported shortly?

Reply:
I think adding such a note makes sense. @aaltay Any thoughs?